main.rb

```diff
@@ -0,0 +1,149 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+def create_student(age, name)
+  print('Has Parent Permission [y/n]:')
+  permission = gets.chomp
+  if permission == 'y'
+    Student.new(age, name)
+  else
+    Student.new(age, name, parent_permission: false)
+  end
+end
+
+def create_teacher(age, name)
+  print('Specialization:')
+  specialization = gets.chomp
+  Teacher.new(age, specialization, name)
+end
+
+def create_person
+  print('Do you want to create a student(1) or a Teacher (2) [Input the number]:')
```

Comment:
- [ ] The text _Teacher_ should be **in lowercase**, as can be seen from the given example. Kindly change the text from _Teacher_ to _teacher_.

Reply:
Fixed